aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -329,49 +333,61 @@ impl AptosVM {
         Ok(())
     }
 
+    /// Execute all module initializers.
     fn execute_module_initialization<S: MoveResolverExt>(
         &self,
         session: &mut SessionExt<S>,
         gas_status: &mut GasStatus,
-        modules: &ModuleBundle,
+        modules: &[CompiledModule],
         senders: &[AccountAddress],
     ) -> VMResult<()> {
         let init_func_name = ident_str!("init_module");
```

Comment:
nit: Not introduced in this PR but I think this should have been made a constant.